# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/bicon/02-hotfix-variable-definiton.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

diff --git a/bicon/bicon_read.h b/bicon/bicon_read.h
index 1956364..66bbe3c 100644
--- a/bicon/bicon_read.h
+++ b/bicon/bicon_read.h
@@ -1,7 +1,7 @@
 #ifndef __BICON_READ_H
 #define __BICON_READ_H
 
-int bicon_options;
+extern int bicon_options;
 
 ssize_t bicon_read (
   int fd,
